homeassistant/components/climate/const.py

```diff
@@ -17,6 +17,9 @@ class HVACMode(StrEnum):
     # All activity disabled / Device is off/standby
     OFF = "off"
 
+    # Device is full on
+    ON = "on"
```

Comment:
Please don't change the climate integration.

Reply:
That's just add the posibility to have "on" mode status... I think there is not only tuya who have this mode ?

Reply:
Changing our entity model needs approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

There's already at least one similar existing discussion. If you want to wait for that please close this PR. Otherwise I suggest you take a different approach that doesn't need changes to our climate entity model.